### PR TITLE
Updated the README.md

### PR DESCRIPTION
**Pull request by aym-i**
Hi!

I updated Godot's README.md, from the looks to the wording in some sentences.

And i also organized it a bit for a friendly UX.

(Please add the hacktoberfest badge to this pr/mr)

Hope you like it!

### CODE REVIEW

**Comment by Calinou** [comment]
These changes are not desired for the most part. They add HTML that makes the file more difficult to read in a text editor.

File: README.md [hunk 1]
```diff
@@ -1,77 +1,82 @@
...
 number of platforms, including the major desktop platforms (Linux, macOS,
 Windows), mobile platforms (Android, iOS), as well as Web-based platforms
-and [consoles](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html).
+and [consoles ↗](https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html).
 
-## Free, open source and community-driven
+## 💸 Free, open source and 🚻 community-driven
 
-Godot is completely free and open source under the very permissive [MIT license](https://godotengine.org/license).
+- Godot is completely **Free** and **Open source** under the free of charge [MIT license ↗](https://godotengine.org/license).
```
Comment (Calinou) [inline README.md:29]: MIT isn't just a "free of charge" license – it is a very permissive license.

File: README.md [hunk 2]
```diff
@@ -1,77 +1,82 @@
...
 No strings attached, no royalties, nothing. The users' games are theirs, down
 to the last line of engine code. Godot's development is fully independent and
 community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the [Software Freedom Conservancy](https://sfconservancy.org/)
+expectations. It is also supported by the [Software Freedom Conservancy ↗](https://sfconservancy.org/)
```
Comment (Calinou) [inline README.md:33]: "also" is not needed here.

File: README.md [hunk 3]
```diff
@@ -1,77 +1,82 @@
...
-Before being open sourced in [February 2014](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac),
-Godot had been developed by [Juan Linietsky](https://github.com/reduz) and
-[Ariel Manzur](https://github.com/punto-) (both still maintaining the project) for several
+> Before being open sourced in [February 2014 ↗](https://github.com/godotengine/godot/commit/0b806ee0fc9097fa7bda7ac0109191c9c5e0a1ac),
+Godot had been developed by [Juan Linietsky ↗](https://github.com/reduz) and
+[Ariel Manzur ↗](https://github.com/punto-) (both still maintaining the project) for several
 years as an in-house engine, used to publish several work-for-hire titles.
 
-![Screenshot of a 3D scene in the Godot Engine editor](https://raw.githubusercontent.com/godotengine/godot-design/master/screenshots/editor_tps_demo_1920x1080.jpg)
+![Screenshot of a 3D scene in the Godot Engine editor ↗](https://raw.githubusercontent.com/godotengine/godot-design/master/screenshots/editor_tps_demo_1920x1080.jpg)
```
Comment (Calinou) [inline README.md:41]: This is image alt text – it doesn't need an external link icon.

File: README.md [hunk 4]
```diff
@@ -1,77 +1,82 @@
...
 provided by the community, such as text and video tutorials, demos, etc.
-Consult the [community channels](https://godotengine.org/community)
+Consult the [community channels ↗](https://godotengine.org/community)
 for more information.
 
-[![Code Triagers Badge](https://www.codetriage.com/godotengine/godot/badges/users.svg)](https://www.codetriage.com/godotengine/godot)
-[![Translate on Weblate](https://hosted.weblate.org/widgets/godot-engine/-/godot/svg-badge.svg)](https://hosted.weblate.org/engage/godot-engine/?utm_source=widget)
-[![Total alerts on LGTM](https://img.shields.io/lgtm/alerts/g/godotengine/godot.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/godotengine/godot/alerts)
-[![TODOs](https://badgen.net/https/api.tickgit.com/badgen/github.com/godotengine/godot)](https://www.tickgit.com/browse?repo=github.com/godotengine/godot)
+## 🖼 Demos
```
Comment (Calinou) [inline README.md:79]: Missing line break below heading.

File: README.md (hunk 3 above)
Comment (retr0cube) [reply README.md:41]: Oops my bad

**Comment by akien-mga** [comment]
Please read the pinned issue: https://github.com/godotengine/godot/issues/66666

This is explicitly the kind of PR we don't want from new contributors.

**Pull request closed without merging**